extension/popup/popupController.jsx

```diff
@@ -236,9 +326,19 @@ export const PopupController = function() {
   };
 
   const setPopupView = async newView => {
+    // do not change view if timer elapsed; timer has highest priority
+    if (timerElapsed === true) {
+      return;
+    }
```

Comment:
This also doesn't seem right... but if we allow the recorder to run and treat the timer as an additional widget on the popup then I think this will be alright. That is, timer shouldn't be a "view", it'll just be an extra thing we show so long as the timer exists. Then there's still room for a card or anything else (the popup might get a little long, but that seems better than disabling functionality while the timer runs).

Reply:
@ianb the recorder works while the timer is active; I am not sure what should be changed right now; I will push the changes requested shortly and let me know if the timer doesn't work right.